arch/arm64/src/Toolchain.defs

```diff
@@ -38,14 +38,18 @@ ifeq ($(CONFIG_ARCH_ARMV8R),y)
   ARCHCPUFLAGS += -march=armv8-r
 endif
 
+
 ifeq ($(CONFIG_ARCH_CORTEX_A53),y)
   ARCHCPUFLAGS += -mcpu=cortex-a53
+else ifeq ($(CONFIG_ARCH_CORTEX_A55),y)
+  ARCHCPUFLAGS += -mtune=cortex-a55
```

Comment:
why not -mcpu like the other cortex-a chips? Check out how this file looks like in upstream; I think this was managed somehow differently there